ResearchKit/Common/ORKPasscodeViewController.h

```diff
@@ -80,12 +80,46 @@ ORK_CLASS_AVAILABLE
 
 - (instancetype)init NS_UNAVAILABLE;
 
+/**
+ An authenticaiton passcode view controller allows a user to be authenticated using
+ the passcode stored in the keychain.
+ 
+ You must check to see if the passcode is stored in the keychain before presenting
```

Comment:
Can you mention `isPasscodeStoredInKeychain` in this doc? And refers passcodeStep for creation as well?